vault/logical_raw.go

```diff
@@ -110,11 +141,83 @@ func (b *RawBackend) handleRawWrite(ctx context.Context, req *logical.Request, d
 		}
 	}
 
-	value := data.Get("value").(string)
+	v := data.Get("value").(string)
+	value := []byte(v)
+	if encoding == "base64" {
+		var err error
+		value, err = base64.StdEncoding.DecodeString(v)
+		if err != nil {
+			return logical.ErrorResponse(err.Error()), logical.ErrInvalidRequest
+		}
+	}
+
+	if req.Operation == UpdateCapability {
```

Comment:
```suggestion
	if req.Operation == logical.UpdateOperation {
```

Reply:
I'm not sure this test actually works.  All PUT/POSTs are "update"s by default, and we only distinguish create/update if the backend defines an ExistenceCheck, which I don't think we have for sys/raw.

Reply:
I defined an existence check.   lines 282-290 of the latest revision.

Reply:
My bad, should've kept reading.

Reply:
Not resolving this conversation yet because I still want to see us use the correct symbol to compare against (even if they both have the value "update" so it doesn't have any code impact, just a readability thing.)

Reply:
using the correct symbol